static/app/components/gridEditable/index.tsx

```diff
@@ -98,23 +98,24 @@ type GridEditableProps<DataRow, ColumnKey> = {
   bodyStyle?: React.CSSProperties;
   emptyMessage?: React.ReactNode;
   error?: unknown | null;
+  fitMaxContent?: boolean;
```

Comment:
This API will restrict you from having different fit options later on. It would be preferable to use a string literal here as opposed to bool, that way you preserve some flexibility for the future.
```suggestion
  fit?: "max-content";
```